.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.10.0
+  - 1.18.0
```

Comment:
I guess that the whole point of this travis entry was to make check for backwards compatibility with older rust versions (not set check with the latest stable 😄 because it is done in travis entry below)

ease note that "?" became stable in 1.13.0. @burntsushi will have to decide if he wants to drop backwards compatibility with 1.10 for the sake of "?" in examples. Otherwise the `try!` macro would have to be used instead.